Dockerfile

```diff
@@ -31,16 +38,19 @@ for pkg in ${PKGS_TO_INSTALL}; do\n\
   }\n\
 done\n\
 yum -y install ${PKGS_TO_INSTALL}\n' >> /tmp/safe_yum.sh \
-&& /bin/bash /tmp/safe_yum.sh \
-&& yum clean all
+  && /bin/bash /tmp/safe_yum.sh \
+  && yum clean all \
+  && rm /tmp/safe_yum.sh
 
 RUN wget https://bootstrap.pypa.io/get-pip.py \
-  && python3.6 get-pip.py
+  && python3.6 get-pip.py \
+  && pip install -U pip pipenv uwsgi \
+  && rm -rf ~/.cache/pip
 
-#
-# This is needed by click: http://click.pocoo.org/5/python3/
-#
-ENV LC_ALL=en_US.UTF-8
-ENV LANG=en_US.UTF-8
+ENV LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    APPLICATION_ROOT="/"
```

Comment:
We can move this `ENV` declaration up as well unless an intermediary step might override any of these values.

Reply:
Fixed in 67927e7